metadata.json

```diff
@@ -99,12 +99,12 @@
     },
     {
       "name": "puppet",
-      "version_requirement": "3.x"
+      "version_requirement": "3.5"
```

Comment:
hm ok don't know its broken, i depend on it because for validate_cmd minimal version is 3.5